strawberry_django/tests.py

```diff
@@ -0,0 +1,52 @@
+import json
+from typing import Optional
+
+from django.test import Client, TestCase
+
+
+class StrawberryTestCase(TestCase):
+    """
+    Based on: https://www.sam.today/blog/testing-graphql-with-graphene-django/ and https://docs.graphene-python.org/projects/django/en/latest/testing/
+    """
+
+    def setUp(self):
+        self._client = Client()
```

Comment:
Could we use strawberry's test client in here? https://github.com/strawberry-graphql/strawberry/blob/main/strawberry/django/test/client.py#L6